SpatialGDK/Source/SpatialGDK/Private/EngineClasses/SpatialActorChannel.cpp

```diff
@@ -421,6 +442,25 @@ FHandoverChangeState USpatialActorChannel::CreateInitialHandoverChangeState(cons
 	return HandoverChanged;
 }
 
+void USpatialActorChannel::GetLatestAuthorityChangeFromHierarchy(const AActor* RootActor, uint64& OutTimestamp)
```

Comment:
I don't think `RootActor` is a good name here. It's true for when it's called directly from `ReplicateActor` but then we recursively call it on all Actors in a hierarchy. 

I prefer `HierarchyActor`